dbt/include/athena/macros/materializations/seeds/helpers.sql

```diff
@@ -64,7 +64,7 @@
           {%- set type = column_override.get(col_name, inferred_type) -%}
           {%- set type = type if type != "string" else "varchar" -%}
           {%- set column_name = (col_name | string) -%}
-          {{ adapter.quote_seed_column(column_name, quote_seed_column) }} {{ ddl_data_type(type) }} {%- if not loop.last -%}, {% endif -%}
+          {{ adapter.quote_seed_column(column_name, quote_seed_column, "`") }} {{ ddl_data_type(type) }} {%- if not loop.last -%}, {% endif -%}
```

Comment:
Should this also apply for `create table as` or not ?

Reply:
weirdly not no. Its only `create external table` which insists on the backticks around underscore-prefixed column names.